tests/test_modeling_common.py

```diff
@@ -395,10 +400,31 @@ def _create_and_check_torchscript(self, config, inputs_dict):
                     attention_mask = inputs["attention_mask"]
                     decoder_input_ids = inputs["decoder_input_ids"]
                     decoder_attention_mask = inputs["decoder_attention_mask"]
-
-                    traced_model = torch.jit.trace(
```

Comment:
Let's try to not change this test. `head_mask` is more or less never used in `torch.jit.trace`. If you're keen we could overwrite this test in all Bart-like models including the head_mask, but it's not mandatory at all IMO.

However, we don't really like these `if model.config.model_type not in ....` statements in the tests.